### ISSUE TITLE
hope jsonable_encoder can parse numpy

### ISSUE THREAD

**Issue by stringk245** [tiangolo/fastapi#2222]
import numpy as np
number = np.array([1, 2, 3, 4])
number = jsonable_encoder(number)  # i hope jsonable_encoder can do, not  number.tolist()  or  int(np.int64(10)) 
response={'number':number}
return response

**Comment by ycd** [comment]
I do not think we can implement this to FastAPI, Because our current encoder implementation does this.

```python
isinstance(obj, class)
``` 
https://github.com/tiangolo/fastapi/blob/dca9cc3ec516b048fadfa42778a7763ad277c7ae/fastapi/encoders.py#L62-L67

If we are going to do that to `numpy.array`, which will look something like this.

```python
if isinstance(obj, np.array)
```

We need to import the whole NumPy, which is very ineffective and unrelated imho,  because NumPy is quite a large library to adapt to a micro-framework like FastAPI, even if it was a small package i do not think this should be inside FastAPI.

**Comment by stringk245** [comment]
........................tks  ........................

if  i get  a return result is big dict like:
result = {'a1':1,........, 'number':np.array([1, 2, 3, 4]), 'a50':50 }   # someone return 
and i need return to  fastapi Response
if i Direct return this result  will be  error, so i hope   jsonable_encoder(result )  can do ,
Rather than always i do like:
for k in result:
    if isinstance(result[k],np.array):
          result[k] = result[k].tolist()
then
return  result to fastapi  Response

**Comment by tiangolo** [comment]
Thanks for the help @ycd! 🚀 

I imagine you solved your problem @stringk245, thanks for closing the issue!

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
